### PR TITLE
use correct branches for the datajam

### PR DESCRIPTION
**Pull request by mulby**
Reviewers: @rocha, @wedaly, @brianhw

### CODE REVIEW

File: playbooks/roles/datajam/files/edx-cms-devserver [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd /edx/app/edxapp/edx-platform
+./manage.py cms --settings devstack runserver --traceback 0.0.0.0:8001
```
Comment (brianhw) [inline playbooks/roles/datajam/files/edx-cms-devserver]: Is line-ending an issue?  Looks like a number of devserver scripts are missing the last linefeed.

Comment (mulby) [inline playbooks/roles/datajam/files/edx-cms-devserver]: oops, I'll add those in.  Carlos and I also discussed replacing these with upstart scripts - which would probably be better.

**Comment by brianhw** [comment]
LGTM :+1:

**Comment by rocha** [comment]
:rocket:

**Pull request merged**